src/frontend/apps/lti_site/components/LiveInfoBar/index.tsx

```diff
@@ -37,13 +45,13 @@ export const LiveInfoBar = ({ title, startDate }: LiveInfoBarProps) => {
       </Heading>
 
       <Box direction="row">
-        {participants.length > 0 && (
+        {numberOfStudents > 0 && (
           <Paragraph
             color="blue-active"
             margin={{ right: 'large', bottom: 'none' }}
             size="small"
           >
-            {`${participants.length} viewers connected.`}
+            <FormattedMessage values={{ count: numberOfStudents }} {...messages.nbViewers}/>
```

Comment:
you should use `intl.formatMessage` instead of `FormattedMessage` component here.

https://formatjs.io/docs/react-intl/api#formatmessage

Reply:
All good ! I added a test for it